src/java/org/apache/cassandra/repair/RepairSession.java

```diff
@@ -114,13 +115,13 @@
 
     /**
      * Create new repair session.
-     *
-     * @param parentRepairSession the parent sessions id
+     *  @param parentRepairSession the parent sessions id
      * @param id this sessions id
      * @param ranges ranges to repair
      * @param keyspace name of keyspace
      * @param parallelismDegree specifies the degree of parallelism when calculating the merkle trees
      * @param endpoints the data centers that should be part of the repair; null for all DCs
```

Comment:
Data centers? This comment is entirely wrong. Also this naming for endpoints ambiguous/misleading